Apps/Admin/Tests/Functional/cypress/integration/e2e/communications.cy.js

```diff
@@ -1,8 +1,26 @@
 function getTodayPlusDaysDate(days) {
-    let ms = new Date(Date.now());
-    ms.setDate(ms.getDate() + days);
-    // yyyy-mm-dd
-    return ms.toISOString().substring(0, 10);
+    let newDay = new Date(Date.now());
+    newDay.setDate(newDay.getDate() + days);
+
+    // Convert the date and time to a localized string
+    const localizedDateString = newDay.toLocaleString("en-US", {
+        timeZone: "America/Vancouver",
+        hour12: false,
+    });
+
+    // Parse the localized string to a Date object
+    const parsedDate = new Date(localizedDateString);
+    cy.log(`Parsed date: ${parsedDate}`);
+
+    // Get the components: year, month, and day
+    const year = parsedDate.getFullYear();
+    const month = String(parsedDate.getMonth() + 1).padStart(2, "0");
+    const day = String(parsedDate.getDate()).padStart(2, "0");
+
+    // Format the date to "yyyy-mm-dd" format
+    const today = `${year}-${month}-${day}`;
+    cy.log(`Today plus ${days} => ${today}`);
+    return today;
```

Comment:
Nit: the date won't be today if any days were added.

```suggestion
    const dateString = `${year}-${month}-${day}`;
    cy.log(`Today plus ${days} => ${dateString}`);
    return dateString;
```